resources/assets/js/vendor.js

```diff
@@ -4,7 +4,3 @@ window.bootstrap = require('bootstrap');
 require('./forms');
 require('./sticky-headers');
 require('./countdown');
```

Comment:
`import` instead of `require`?

Reply:
Out of scope for this PR. https://github.com/engelsystem/engelsystem/issues/1006